src/staticanalysis/bot/default.nix

```diff
@@ -51,7 +51,6 @@ let
           env = {
             "SSL_CERT_FILE" = "${cacert}/etc/ssl/certs/ca-bundle.crt";
             "APP_CHANNEL" = branch;
-            "MOZ_AUTOMATION" = "1";
```

Comment:
Are you sure that does not break other parts ? It was needed to bypass a `mach` check for staticanalysis